doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -120,9 +120,27 @@ PSI Variables by Alpha
    energy [H] and correlation correction components [H] for the compound
    method requested through cbs().
 
+.. psivar:: CC ROOT n DIPOLE X
```

Comment:
is it necessarily duplicated? If some sort of `FOLLOW_ROOT` option was set to `2`, then `CC DIPOLE X` would equal `CC ROOT 2 DIPOLE X`. If you're wanting to mention that general duplication, I agree.